E2E/tests/copyLink.test.ts

```diff
@@ -0,0 +1,17 @@
+Feature('Copy link');
```

Comment:
This seems to be missing test that would check if the link was actually copied into the cliipboard.